platform/ios/scripts/package.sh

```diff
@@ -270,5 +271,7 @@ sed -i '' \
     "${README}"
 cp ${README} "${OUTPUT}"
 
-step "Generating API documentation…"
-make idocument OUTPUT="${OUTPUT}/documentation"
+if [ ${SKIP_DOCS} == false ]; then
```

Comment:
This is a bit of a double negative. I think `${BUILD_DOCS} == true` would be easier to grok, but it’s no big deal.

Reply:
I'm trying to be less negative in general, this is a good first step.